bool is_filtering_all_messages
---
bool ok
